streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -71,6 +71,7 @@
 import static org.apache.kafka.common.config.ConfigDef.Range.between;
 import static org.apache.kafka.common.config.ConfigDef.ValidString.in;
 import static org.apache.kafka.common.config.ConfigDef.parseType;
+import static org.apache.kafka.streams.internals.UpgradeFromValues.UPGRADE_FROM_35;
```

Comment:
After merging and doing the follow up PR to add `3.6`, I realized that using this import is not what we want to do.

Call for review (follow up cleanup PR): https://github.com/apache/kafka/pull/15208